cfgov/login/urls.py

```diff
@@ -14,56 +12,60 @@
         # If SAML2 auth is enabled, /login will redirect to /saml2/login,
         # which in turn will redirect to the configured identity provider.
         re_path(r"saml2/", include("djangosaml2.urls")),
-        # For backup/admin/emergency auth in the event that SSO is down,
-        # the Django login is available at /django-admin/login. Wagtail's
-        # /admin/login is redirected to /login for both SSO and non-SSO auth.
+        # Redirect the Wagtail login/logout views to djangosaml2.
+        # For admin authentication, /django-admin/login is preserved when SSO
+        # is enabled.
         re_path(
-            r"^login/$",
-            RedirectView.as_view(url="/saml2/login/", query_string=True),
+            r"^/admin/login/$",
```

Comment:
Can we `reverse_lazy("wagtailadmin_login")` here, and something similar below for the logout URL?

Reply:
So, no, it looks like the path match is evaluated when the URLs are loaded, and `"wagtailadmin_login"` (for example here) isn't resolvable at that point in time. It is a shame.